src/napari_live_recording/processing_engine_/processing_gui.py

```diff
@@ -0,0 +1,513 @@
+        self.loadExistingFilter_btn = QPushButton("Show Exisiting Filters")
+        self.loadExistingFilter_btn.clicked.connect(self.showExistingFilters)
+        # self.filterNameLineEdit.textChanged.connect(self.handleApply_btn)
+
+        self.clear_btn.clicked.connect(self.clearListWidget)
+        self.createFilter_btn.clicked.connect(self.returnRightListContent)
+        self.rightList.itemDoubleClicked.connect(self.openParameterDialogWindow)
+        # self.apply_btn.setDisabled(True)
+        self.rightContainerLayout.addWidget(self.loadExistingFilter_btn, 0, 0, 1, 2)
+        self.rightContainerLayout.addWidget(self.rightList, 1, 0, 1, 2)
+        self.rightContainerLayout.addWidget(self.clear_btn, 3, 0)
+        self.rightContainerLayout.addWidget(self.createFilter_btn, 3, 1)
+        self.rightContainerLayout.addWidget(self.filterNameLabel, 2, 0)
+        self.rightContainerLayout.addWidget(self.filterNameLineEdit, 2, 1)
+
+        self.previewContainer = QGroupBox()
+        self.previewContainerLayout = QGridLayout()
+        self.previewContainer.setLayout(self.previewContainerLayout)
+        self.refresh_btn = QPushButton("Refresh")
+        self.loadNewPreviewImage_btn = QPushButton("Load new Image")
+
+        self.refresh_btn.clicked.connect(self.updatePreviewImage)
+        self.loadNewPreviewImage_btn.clicked.connect(self.loadPreviewImage)
+        # self.image_ = cv.imread(
+        #     r"C:\git\napari-live-recording\src\napari_live_recording\processing_engine_\testImage.jpg"
+        # )
+        # aspectRatio = self.image_.shape[0] / self.image_.shape[1]
+        # self.image = cv.resize(
+        #     self.image_,
+        #     dsize=(int(280 * aspectRatio), 400),
+        #     interpolation=cv.INTER_CUBIC,
+        # )
+
+        self.imageView = ImageView()
+        # self.imageView.ui.histogram.hide()
+        self.imageView.ui.roiBtn.hide()
+        self.imageView.ui.menuBtn.hide()
+        self.loadPreviewImage(True)
+        #        self.imageView.setImage(self.loadPreviewImage(True))
+        self.previewContainerLayout.addWidget(self.imageView, 0, 0, 1, 2)
+        self.previewContainerLayout.addWidget(self.refresh_btn, 1, 0, 1, 1)
+        self.previewContainerLayout.addWidget(self.loadNewPreviewImage_btn, 1, 1, 1, 1)
+
+        layout = QHBoxLayout(self)
+        layout.addWidget(self.leftContainer)
+        layout.addWidget(self.rightContainer)
+        layout.addWidget(self.previewContainer)
+
+
+if __name__ == "__main__":
+    # don't auto scale when drag app to a different monitor.
+    # QGuiApplication.setHighDpiScaleFactorRoundingPolicy(Qt.HighDpiScaleFactorRoundingPolicy.PassThrough)
+
+    app = QApplication(sys.argv)
+    app.setStyleSheet(
+        """
+        QWidget {
+            font-size: 15px;
+        }
+    """
+    )
+
+    myApp = FilterSelectionWidget()
+    myApp.show()
+
+    try:
+        sys.exit(app.exec())
+    except SystemExit:
+        print("Closing Window...")
```

Comment:
are you sure that isn't there another way to pop up the GUI screen from the existing plugin code?